mypyc/test-data/run-strings.test

```diff
@@ -518,6 +518,18 @@ def test_format_method_python_doc() -> None:
 
 def test_chr() -> None:
     assert chr(57344) == '\ue000'
+    assert chr(0) == '\x00'
```

Comment:
Also test a regular ascii value (e.g. `chr(65)`) and a value between 128 and 255.